tests/e2e/vsphere_volume_expansion.go

```diff
@@ -2387,8 +2387,12 @@ func increaseOnlineVolumeMultipleTimes(ctx context.Context, f *framework.Framewo
 	var err error
 	//Fetch original FileSystemSize
 	ginkgo.By("Verify filesystem size for mount point /mnt/volume1 before expansion")
-	originalSizeInMb, err = getFSSizeMb(f, pod)
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	if windowsEnv {
+		originalSizeInMb = getWindowsPodSize(client, pod)
+	}else {
```

Comment:
run `make check`
add a space after '}'

Reply:
marunachala-a01:vsphere-csi-driver marunachalam$ make check
hack/check-format.sh
hack/check-mdlint.sh
hack/check-shell.sh
hack/check-staticcheck.sh
hack/check-vet.sh
hack/check-golangci-lint.sh
golangci/golangci-lint info checking GitHub for tag 'v1.40.1'
golangci/golangci-lint info found version: 1.40.1 for v1.40.1/darwin/amd64
golangci/golangci-lint info installed /Users/marunachalam/go/bin/golangci-lint
INFO [config_reader] Config search paths: [./ /Users/marunachalam/Desktop/CSI/repo/windows_pr_1/vsphere-csi-driver /Users/marunachalam/Desktop/CSI/repo/windows_pr_1 /Users/marunachalam/Desktop/CSI/repo /Users/marunachalam/Desktop/CSI /Users/marunachalam/Desktop /Users/marunachalam /Users /] 
INFO [config_reader] Used config file .golangci.yml 
INFO [lintersdb] Active 12 linters: [deadcode errcheck gosimple govet ineffassign lll misspell staticcheck structcheck typecheck unused varcheck] 
INFO [loader] Go packages loading at mode 575 (name|types_sizes|exports_file|deps|files|imports|compiled_files) took 2.220409919s 
INFO [runner/filename_unadjuster] Pre-built 0 adjustments in 123.951071ms 
INFO [linters context/goanalysis] analyzers took 18.87513419s with top 10 stages: buildir: 4.927006619s, misspell: 1.286827344s, S1038: 1.240764594s, S1039: 840.49895ms, SA1012: 705.837411ms, S1028: 651.066399ms, S1019: 461.810894ms, lll: 449.47507ms, S1024: 412.925912ms, S1030: 354.916344ms 
INFO [runner] Issues before processing: 111, after processing: 0 
INFO [runner] Processors filtering stat (out/in): cgo: 111/111, autogenerated_exclude: 22/111, identifier_marker: 22/22, nolint: 0/1, path_prettifier: 111/111, skip_files: 111/111, skip_dirs: 111/111, exclude: 22/22, exclude-rules: 1/22, filename_unadjuster: 111/111 
INFO [runner] processing took 20.336503ms with stages: nolint: 15.385256ms, autogenerated_exclude: 2.138733ms, path_prettifier: 1.723142ms, skip_dirs: 515.732µs, identifier_marker: 386.735µs, exclude-rules: 147.79µs, cgo: 18.031µs, filename_unadjuster: 14.019µs, max_same_issues: 1.683µs, uniq_by_line: 1.428µs, skip_files: 552ns, source_code: 505ns, max_from_linter: 503ns, exclude: 486ns, diff: 446ns, path_shortener: 359ns, severity-rules: 307ns, sort_results: 307ns, max_per_file_from_linter: 304ns, path_prefixer: 185ns 
INFO [runner] linters took 9.331940544s with stages: goanalysis_metalinter: 9.311411919s 
INFO File cache stats: 83 entries of total size 2.1MiB 
INFO Memory: 118 samples, avg is 336.8MB, max is 615.0MB
INFO Execution took 11.691124658s